en/README.md

```diff
@@ -0,0 +1,1138 @@
+|:-----------------|:--|:-:|:----------:|:---|
+|childHostId |String255|M|-|Child host id.|
+|parentHostId|String255|M|-|Parent host id.|
+
+```json
+{
+  "id": 1,
+  "params": {
+    "lastInfo": "201504152246",
+    "updateType": "ALL",
+    "hostParents": [
+      {
+        "parentHostId": "10",
+        "childHostId": "12"
+      },
+      {
+        "parentHostId": "20",
+        "childHostId": "11"
+      }
+    ]
+  },
+  "method": "putHostParent",
+  "jsonrpc": "2.0"
+}
+```
+
+***Result(result)***
+
+Receive result as result object value whether sent data has been updated. Look at the [[Values](#user-content-putresult)].
+
+```json
+{
+  "id": 1,
+  "result": "SUCCESS",
+  "jsonrpc": "2.0"
+}
+```
+
+### putArmInfo(method)
+
+Standard motion is to send after each to send host, trigger, event information to Hatohol server, but can send anytime. The minimum interval is 1 second(MUST), The maximum interval is twice of polling interval second that got by the [getMonitoringServerInfo](#user-content-getmonitoringserverinfomethod) and [updateMonitoringServerInfo](#user-content-updatemonitoringserverinfomethod)(SHOULD).
+
+***Request(params)***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|lastStatus         |string   |M|-|Latest polling result. [[$B0lMw(B](#user-content-arminfostatus)]|
+|failureReason      |String255|M|-|Reason of fail to get information. |
+|lastSuccessTime    |TimeStamp|M|-|Time of latest success to get information. If have not been successful once, set empty string.|
+|lastFailureTime    |TimeStamp|M|-|Time of latest failure to get information. If have not been unsuccessful once, set empty string.|
+|numSuccess         |Number   |M|-|The number of success to get information from start up HAP.|
+|numFailure         |Number   |M|-|The number of failure to get information from start up HAP.|
+
+```json
+{
+  "id": 1,
+  "params": {
+    "numFailure": 10,
+    "numSuccess": 165,
+    "lastFailureTime": "20150313161500",
+    "lastSuccessTime": "20150313161100",
+    "failureReason": "Example reason",
+    "lastStatus": "INIT"
+  },
+  "method": "putArmInfo",
+  "jsonrpc": "2.0"
+}
+```
+
+***Result(result)***
+
+Receive result as result object value whether sent data has been updated. Look at the [[Values](#user-content-putresult)].
+
+```json
+{
+  "id": 1,
+  "result": "SUCCESS",
+  "jsonrpc": "2.0"
+}
+```
+
+### fetchItems(method)
+
+This procedure request to get item to HAP. HAP must return success or failure of to accept a that request. After that, Send to matched items using a putItems procedure. At that time, need to pass fetchId and hostIds of fetchItems procedure params to putItems procedure.
+
+***Request(params)***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|hostIds|String255 array|O|-|Get triggers of to designate a hosts. If does not designate hostIds, all trigger are targeted.|
```

Comment:
> Get triggers of to designate a hosts.

Narrow down host(s) to obtain triggers.

> If does not designate hostIds, all trigger are targeted.

If not specifying hostId(s), all trigger should be subject to this procedure.